hubblestack/extmods/fdg/stat.py

```diff
@@ -0,0 +1,264 @@
+'''
```

Comment:
I think even the module doc should start with `"""` rather than `'''` … it's a pep8 rule iirc. I don't completely agree with it, personally, but it's worth conforming to something for consistency's sake.

Reply:
Done.